src/utils/searchAlgorithm.js

```diff
@@ -1,7 +1,10 @@
 import { matchSorter } from 'match-sorter'
-import colleges from '../../data/university_data.json'
 
-const searchAlgorithm = (input) => {
-  return matchSorter(colleges, input, { keys: ['name', 'address'] })
+const searchAlgorithm = (input, data) => {
+  if (input && data) {
+    return matchSorter(data, input, { keys: ['name', 'country', 'city'] })
+  } else {
+    return []
```

Comment:
When I checked in the codecov, it turns out, this line wasn't tested at the moment. So, let's write a simple test for it, which can be triggered by empty data.